packages/l10n/README.md

```diff
@@ -97,3 +97,265 @@ First, download the data (`core.zip`) for a specific version from the [downloads
+
+```js
+// Product, returned from the `/graphql` API of commercetools platform
+{
+  nameAllLocales: [
+    {
+      locale: 'en',
+      value: 'Milk',
+    },
+  ];
+}
+```
+
+To distinguish these in source code of the Merchant Center, we name the graphql shaped value `LocalizedField`. We offer `applyTransformedLocalizedFields` that is authored to transform these values from one to the other.
+
+#### Example usage
+
+```js
+const product = {
+  nameAllLocales: [
+    {
+      locale: 'en',
+      value: 'Milk',
+    },
+  ],
+  descriptionAllLocales: [
+    {
+      locale: 'en',
+      value: 'This is milk',
+    },
+  ],
+};
+const fieldNameTransformationMappings = [
+  {
+    from: 'nameAllLocales',
+    to: 'name',
+  },
+  {
+    from: 'descriptionAllLocales',
+    to: 'description',
+  },
+];
+const transformedProduct = applyTransformedLocalizedFields(
+  fetchedProduct,
+  fieldNameTransformationMappings
+);
+
+console.log(transformedProduct);
+// { name: { en: 'Milk' }, description: { en:  'This is milk' } }
+```
+
+#### When to use it
+
+This transformation tool will be helpful when you consume the commercetools platform `/graphql` API in conjunction with authoring views consuming `@commercetools-frontend/ui-kit`.
+
+Given that you consume:
+
+- The commercetools platform `/graphql` API
+- [`LocalizedTextInput`](https://github.com/commercetools/ui-kit/blob/master/packages/components/inputs/localized-text-input/src/localized-text-input.js)
+
+This will be helpful transforming data from `response -> view`.
+
+```js
+// fetching product from the commercetools platform `/graphql` API
+// returns a product with a `nameAllLocales` and `descriptionAllLocales`
+const product = useMcQuery(ProductQuery, {
+  context: GRAPHQL_TARGETS.COMMERCETOOLS_PLATFORM,
+});
+
+// Given that LocalizedTextInput accepts a value of `{ [key: string]: string }`,
+// we tranform our product to match the required shape
+const transformedProduct = applyTransformedLocalizedFields(
+  product,
+  fieldNameTransformMappings
+);
+
+// Finally, we are ready to render our form with the correctly shaped `name` and `description`
+return (
+  <>
+    <LocalizedTextInput name="name" value={transformedProduct.name} />
+    <LocalizedTextInput
+      name="description"
+      value={transformedProduct.description}
+    />
+  </>
+);
+```
+
+### `formatLocalizedString` (DRAFT)
+
+> Transforms a `LocalizedString` to a String
+
+The `formatLocalizedString` util is a util we use internally inside the Merchant Center.
+This util is at the moment subject to change, hence marked as Draft. Use with caution in your Custom Application.
+
+#### Context
+
+As discussed under `applyTransformedLocalizedFields`, a `LocalizedString` is a value type reserved for a `Resource`, e.g a `Product`.
+
+```js
+const product = {
+  name: {
+    en: 'Milk',
+    de: 'Milsch,
```

Comment:
```suggestion
    de: 'Milch,
```

Reply:
I'm getting german class... but no remark on the missing `'` :trollface: